Library/Formula/osquery.rb

```diff
@@ -52,9 +53,9 @@ def install
   test do
     require 'open3'
     Open3.popen3("#{bin}/osqueryi") do |stdin, stdout, _|
-      stdin.write(".mode line\nSELECT major FROM osx_version;")
+      stdin.write(".mode line\nSELECT version FROM osquery_info;")
       stdin.close
-      assert_equal "major = 10\n", stdout.read
+      assert_equal "version = #{version}\n", stdout.read
```

Comment:
The concern is our `version` may not always match `osquery`'s version here and we'd rather not have to change this each time.